dev_tools/utils/nxdl_utils.py

```diff
@@ -232,7 +234,7 @@ def belongs_to_capital(params):
     (act_htmlname, chk_name, name_any, nxdl_elem, child, name) = params
     # or starts with capital and no reserved words used
     if (
-        (name_any or "A" <= act_htmlname[0] <= "Z")
+        (act_htmlname[0].isalpha() and act_htmlname[0].isupper())
```

Comment:
LHS and RHS are functionally not necessarily equivalent,
what if name_any (on the left-hand side / LHS) is boolean, also brackets should be used to clarify operator precedence
lexicographical ordering https://stackoverflow.com/questions/4806911/how-are-strings-compared

Reply:
Thanks! Indeed, it is a mistake, because name_any property shall also be considered here!